src/core_plugins/kibana/public/discover/index.html

```diff
@@ -142,11 +142,9 @@
               ></doc-table>
 
               <div ng-if="rows.length == opts.sampleSize" class="discover-table-footer">
-                <center>
-                  These are the first {{opts.sampleSize}} documents matching
-                  your search, refine your search to see others.
-                  <a ng-click="toTop()" title="Back to top">Back to top.</a>
-                </center>
+                These are the first {{opts.sampleSize}} documents matching
+                your search, refine your search to see others.
+                <a ng-click="toTop()">Back to top.</a>
```

Comment:
Not accessibility related, just removed the `<center>` tag and changed the CSS to center this instead.